01_install_requirements.sh

```diff
@@ -11,7 +11,11 @@ sudo sed -i "s/=enforcing/=permissive/g" /etc/selinux/config
 sudo yum -y update
 
 # Install EPEL required by some packages
-sudo yum -y install epel-release --enablerepo=extras
+if grep -q "Red Hat Enterprise Linux" /etc/redhat-release ; then
+    sudo yum -y install http://mirror.centos.org/centos/7/extras/x86_64/Packages/epel-release-7-11.noarch.rpm
```

Comment:
I'd say to better use the latest epel package just in case

```
sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Reply:
sorry forgot to do this will update if I have to do another push